pkg/controller/account/account_controller.go

```diff
@@ -120,9 +135,34 @@ type newAwsClientInput struct {
 func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling Account")
+
+	// We expect this secret to exist in the same namespace Account CR's are created
+	awsSetupClient, err := r.getAWSClient(newAwsClientInput{
+		secretName: awsSecretName,
+		nameSpace:  awsv1alpha1.AccountCrNamespace,
+		awsRegion:  "us-east-1",
+	})
+	if err != nil {
+		reqLogger.Info("Failed to get AWS client")
+		return reconcile.Result{}, err
+	}
+
+	// before doing anything make sure we are not over the limit if we are just error
+	accountTotal, err := TotalAwsAccounts(awsSetupClient)
+	if err != nil {
+		reqLogger.Info(err.Error())
+		return reconcile.Result{}, err
+	}
+
+	if accountTotal >= awsLimit {
+		print(accountTotal)
+		reqLogger.Info("At aws account limit Total Account: %d")
```

Comment:
nit picking, capitals for AWS, this should also be an error

Reply:
Resolving in favor of suggestion in my review.  Thanks for pointing it out :)